tardis/model/base.py

```diff
@@ -84,6 +84,26 @@ def r_outer(self):
         return self.geometry.r_outer.values * self.geometry_units["r_outer"]
 
 
+class Geometry:
```

Comment:
For now, just make a class called `Radial1DGeometry` which will eventually inherit from a Geometry parent class.  The `Radial1DGeometry` should contain the attributes needed to run tardis with a radial 1D geometry